src/main/java/org/entando/kubernetes/client/hub/domain/BundleGroupEntityDto.java

```diff
@@ -0,0 +1,51 @@
+package org.entando.kubernetes.client.hub.domain;
+
+import java.util.HashSet;
+import java.util.Objects;
+import java.util.Set;
+import javax.persistence.CascadeType;
+import javax.persistence.FetchType;
+import javax.persistence.OneToMany;
+import lombok.Data;
+import lombok.ToString;
+
+/**
+ * This entity class is for BUNDLE_GROUP table.
+ */
```

Comment:
please remove this comment because inside the component manager it has no meaning (the CM does not know the db structure)

Reply:
Done